.github/ISSUE_TEMPLATE/new-contribution.md

```diff
@@ -6,7 +6,7 @@ about: Solicit feedback for a contribution you'd like to make
 How exciting!
 
 Please review the following link for information to include in your issue:
-https://docs.opendp.org/en/develop/developer/intro.html#code-and-proof-contributions.html
+https://docs.opendp.org/en/develop/developer/intro.html#code-and-proof-contributions
```

Comment:
I'm confused. Do we really want the "develop" branch here? Shouldn't it be "stable" or "latest"?

Reply:
Nope, we don't want "develop". Crafting these urls by hand is error-prone. Thanks for catching this!